docs/roles-and-responsibilities.md

```diff
@@ -58,7 +58,8 @@ Back-up for Primary Meta Releaser
 
 ## Role: Community Releaser
 
-Member of the React Native Core Contributor community. There are usually 2 Community Releasers per release.
+Member of the React Native Core Contributor community or Frameworks developers.
+There are usually 2+ Community Releasers per release.
```

Comment:
nit: I wouldn't recommend splitting sentences into newlines, this isn't a wide convention and will lead to inconsistencies in our docs. We can rely on the developer using text wrapping within their editor.

Extended: This approach cannot be consistently used in all Markdown constructs (e.g. within a list), and obstructs formatting a paragraph as a unit.